src/averageAges.js

```diff
@@ -37,7 +43,21 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter(
+      (person, index, peopleList) => (
+        peopleList.some(potentialChild => (
+          potentialChild.mother === `${person.name}`
+        ))
+      )
+    )
```

Comment:
There is very bad code readability because of brackets, I recommend optimizing it.

Reply:
2. If you move your shortened return statement in arrow function to a newline, **use round brackets**